packages/edit-site/src/components/global-styles/font-library-modal/index.js

```diff
@@ -45,9 +50,15 @@ function FontLibraryModal( {
 } ) {
 	const { collections, setNotice } = useContext( FontLibraryContext );
 
+	const canUserCreate = useSelect( ( select ) => {
+		const { canUser } = select( coreStore );
+		return canUser( 'create', 'font-families' );
+	} );
```

Comment:
This should really be checking the permission of the `wp_font_face` post type but I am not sure how to do that. Putting `font-families/(?P<font_family_id>[\d]+)/font-faces` in didn't seem to work. Can I just use the post type?

I'm also not too clear on the difference between `canUser` and `useResourcePermissions` 

Same applies to the deletion capability check in `installed-fonts.js`

Reply:
Think of `canUser( requestedAction, resource, id )` as a shortcut to check the "allow headers" for `wp/v2/:resource/:id` (id being optional). so for `create`, it's going to check whether the `POST`  method is  allowed for that resource (using an OPTIONS request).

I don't think we support nested routes like that yet. We could potentially make a change to `core-data` to support it, but do you know which REST API could give us this information? 

useResourcePermissions is just a shortcut to retrieve multiple permissions for a given resource. (It's a shortcut towards multiple canUser calls)

Reply:
`canUser( 'create', 'font-families/123/font-faces' )` would probably work, but you'd need the font family ID.

Reply:
Thanks both, I'll leave it as is for now as font-family is using the same permissions.